content/md/cloud/2-access-flexibility-server.md

```diff
@@ -40,6 +43,36 @@ Once the contacts are created and their keys imported, the project manager can a
 The SSH key access is **not instantaneous**. Initially, the status will be `Creation pending` and then will change to `Activated` once the access is given.
 :::
 
+## Add access to your flexibility instance to an already existing contact from another company.
+
+1. Click on the `Add contact key` button
+1. Select `Select other contact`
+1. Enter the email of the contact, `This contact exists in the portal.` must appear.
+1. Enter the name of the key and the public key.
+1. Click on the `Add contact key` button to validate the creation.
+
+![Add existing contact from other company](../img/add_contact_other_company_flex.png)
+
+## Add access to your flexibility instance to a new contact
+
+Supervisors or Project Managers have the possibility to **create a new contact** from this form. It is possible to attach this contact directly to your company or to declare it as external.
+The external contact will be visible from the contacts of your company and will be removed from your company if it is no longer linked to your company. For example, if you delete its access.
```

Comment:
if they are no longer linked to your company. For example, if you delete their access.